content/guides/component-testing/getting-started.md

```diff
@@ -3,47 +3,51 @@ title: Getting Started
 containerClass: component-testing
 ---
 
-⚠️ The Cypress Component Testing library is in still in **Alpha**. We are rapidly developing and expect that the API may undergo breaking changes. Contribute to its development by submitting feature requests or issues [here](https://github.com/cypress-io/cypress/).
+<alert type="warning">
+
```

Comment:
I actually had removed the carriage returns around the `<alert>` - I didn't realize those were intentional. I'll keep this in mind. I wonder if we can accomplish the additional padding via some styling (assuming that is what the carriage returns are for - could be wrong about this assumption).

Reply:
unfortunately, this has to do with `remarkjs` antics.